src/main/java/com/mongodb/jdbc/MongoSQLDatabaseMetaData.java

```diff
@@ -1030,7 +1030,7 @@ public ResultSet getCrossReference(
                                                 TABLE_NAME, new BsonString(namespace.right())),
                                         new BsonElement(COLUMN_NAME, new BsonString(key)),
                                         new BsonElement(
-                                                KEY_SEQ, new BsonInt32(pos.getAndIncrement())),
+                                                KEY_SEQ, new BsonInt32(pos.incrementAndGet())),
```

Comment:
I did this drive-by fix instead of filing a skip-reason ticket since it is so small and the index methods already have so much blocking them.